features/supplier/view_and_edit_g_cloud_services.feature

```diff
@@ -54,6 +54,30 @@ Scenario: Supplier user can edit the features and benefits of a service
     | field                         | value                                                                                  |
     | Service features and benefits | Service features Feature 1 New Feature 2 Service benefits Benefit 1 Updated Benefit 2  |
 
+Scenario: Supplier user can replace the service definition document of a service
+  When I click the top-level summary table Edit link for the section 'Documents'
+  Then I am on the 'Documents' page
+  And I choose file 'test.pdf' for the field 'serviceDefinitionDocumentURL'
+  And I click 'Save and return to service'
+  Then I am on the 'Test cloud support service' page
+  And I see a success banner message containing 'You’ve edited your service. The changes are now live on the Digital Marketplace.'
+
+Scenario: Supplier user can not replace the service definition document of a service with a non-pdf file
+  When I click the top-level summary table Edit link for the section 'Documents'
+  Then I am on the 'Documents' page
+  And I choose file 'word.docx' for the field 'serviceDefinitionDocumentURL'
+  And I click 'Save and return to service'
+  Then I am on the 'Documents' page
+  And I see a validation message containing 'Your document exceeds the 5MB limit. Please reduce file size.'
```

Comment:
Is this the right validation message?

Reply:
Sorry, no - I fixed this locally this morning but must have forgotten to push it.